content/events/2020-quebeccity/_index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Québec
+date: '2020-06-18'
+end: '2020-06-18'
+location:
+  country: Canada
+  city: Québec
+  url: https://en.wikipedia.org/wiki/Quebec_City
+  venue: TBD
+#map: bouvet-island-map.png
+sched_slug: kubernetescommunitydayquebeccity
+social:
+  #twitter: 'https://twitter.com/Bouvet_3Y0Z'
+sponsors:
+- level: platine
+  orgs:
+  - name: Microsoft
+    logo: sponsors/microsoft.svg
+    url: https://microsoft.com
+- level: or
+  orgs:
+  - name: RedHat
+    logo: sponsors/redhat.svg
+    url: https://redhat.com
+- level: argent
+  orgs:
+  - name: iA
+    logo: sponsors/ia.png
+    url: https://ia.ca/
+- level: bronze
+  orgs:
+  - name: MayaData
+    logo: 
+    url: https://mayadata.io/
+  - name: Lecentre.net
+    logo: sponsors/lecentrenet.png
+    url: http://www.lecentre.net/
+speakers:
+- name: Phippy
+  image: speakers/phippy-color.png
+  twitter: https://twitter.com/cloudnativefdn
+  bio: Phippy is a simple PHP app, trying to find a home in a cloud native world.
+  homepage: https://phippy.io
```

Comment:
This should just get commented out or removed. You don't need to have speakers listed right now.

Reply:
Done, thanks!